src/app/fdctl/run/main.seccomppolicy

```
@@ -29,6 +29,15 @@ write: (or (eq (arg 0) 2)
 # descriptor 3 is always the logfile.
 fsync: (eq (arg 0) logfile_fd)
 
+# supervisor: resume supervisor execution context when returning from
+# a signal handler
+#
+# If the supervisor receives SIGUSR1 or SIGUSR2, rather than terminating
+# the supervisor process, a designated handler function executes.
+# rt_sigreturn resumes the main execution context after the signal
+# handler returns.
+rt_sigreturn
```

I think per Ben's suggestion, it's better to just set SIG_IGN on SIGUSR1 and SIGUSR2 and have the kernel silently swallow them for us, than to enable rt_sigreturn and log the message.

---

Will change, I agree all the tradeoffs here (complexity, security) seem like a lot for a warning message that can probably be safely ignored.